src/tests/issues.targets

```diff
@@ -932,6 +932,9 @@
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/Regression/CLR-x86-JIT/V2.0-Beta2/b353858/b353858/**">
             <Issue>https://github.com/dotnet/runtimelab/issues/155: Thorough checking of invalid inputs</Issue>
         </ExcludeList>
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Regression/CLR-x86-JIT/V2.0-Beta2/b425314/b425314/**">
+            <Issue>https://github.com/dotnet/runtime/issues/107418</Issue>
+        </ExcludeList>
```

Comment:
This should be in a separate ItemGroup like e.g. `<!-- NativeAOT arm32 specific -->` below.

We don't completely disable tests if there is one configuration where things don't work. I believe conditioning on the value of `ControlFlowGuard` property would work.